Initial commit, plus travis-ci and basic (very basic) test case

patrickv83 opened the pull request.

pvinas-novetta committed:
Initial commit, plus travis-ci and basic (very basic) test case

pvinas-novetta committed:
Added abstract Item class, test that it can't be instantiated

pvinas-novetta committed:
Rename items module to item, fix references

pvinas-novetta committed:
Missed a reference

claudecyrill committed:
v1 Class Diagram

Initial version of class diagram; feel free to elaborate and expand on

pvinas-novetta committed:
Remove backup files, update gitignore

patrickv83 committed:
Merge branch 'master' into development

claudecyrill committed:
Class Diagram Update

Defined Character, Player, and Enemy Class

claudecyrill committed:
Merge branch 'development' of https://github.com/patrickv83/demonata into development

claudecyrill committed:
Class Diagram

Minor adjustments to diagram.

jalyxis55 reviewed (commented):
Want to review Item() on Thur (3/22) before approval.

jalyxis55 committed:
Class Diagram Edit

I have made changes to the class diagram. There are still a few concrete
classes that should be added but this is more in Python now.

patrickv83 committed:
Merge branch 'master' into development
